R/import_data.R

```diff
@@ -41,22 +96,27 @@ import_geo_cods <- function(descargar = FALSE) {
 #' @return Una list con las enfermedades y los años disponibles
 #' para su descarga desde los microdatos de SIVIGILA
 #' @examples
+#' \dontrun{
```

Comment:
Based on Hugo's recommendation, we have to change the usage of `dontrun` to `donttest`

Reply:
Done